docs/user/security/session-management.asciidoc

```diff
@@ -0,0 +1,60 @@
+[role="xpack"]
+[[xpack-security-session-management]]
+=== Session management
+
+When you log in to {kib} it creates a session that is used to authenticate subsequent requests to {kib}. A session consists of two components: an encrypted cookie that is stored in your browser, and an encrypted document in a dedicated {es} hidden index. By default, the name of that index is `.kibana_security_session_1` where the prefix is derived from {kib}'s primary `.kibana` index.
+
+When your session expires, or you log out of {kib} explicitly it will invalidate your cookie and remove session information from the index. {kib} also periodically invalidates and removes any expired sessions that weren't invalidated explicitly.
```

Comment:
Suggestion for wording this a bit more consistently:
```suggestion
When you log into {kib}, it creates a session that is used to authenticate subsequent requests. A session consists of two components: an
encrypted cookie that is stored in your browser, and an encrypted document in a dedicated {es} hidden index. By default, the name of that
index is `.kibana_security_session_1`, where the prefix is derived from {kib}'s primary `.kibana` index. If either of these components are
missing, the session is no longer valid.

When you explicitly log out of {kib}, it will delete your session cookie and remove your session document from the index. {kib} also
periodically cleans up the index to remove any expired session documents.
```